uk.co.compucorp.civicrm.civihr/info.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<extension key="uk.co.compucorp.civicrm.civihr" type="module">
```

Comment:
also the extension key , I think it would be better to make it :

uk.co.compucorp.civicrm.civihrcore

and updating the extension directory name to be the same